package.json

```diff
@@ -28,6 +28,7 @@
     "@graphql-tools/merge": "^6.2.4",
     "@graphql-tools/schema": "^6.2.4",
     "@graphql-tools/utils": "^6.2.4",
+    "@opentelemetry/api": "^0.13.0",
```

Comment:
`@opentelemetry/api` version `0.16.0` changes the API significantly, if we do want to get this in we should try to build against the most up to date tracer API and active-context-getting functions they have available (the `getSpan`, `setSpan` stuff)